Frontend/ui-customer/src/Services/API.purs

```diff
@@ -2474,6 +2476,53 @@ instance standardEncodeBusinessHoursResp :: StandardEncode BusinessHoursResp whe
 instance showBusinessHoursResp :: Show BusinessHoursResp where show = genericShow
 instance decodeBusinessHoursResp :: Decode BusinessHoursResp where decode = defaultDecode
 instance encodeBusinessHoursResp :: Encode BusinessHoursResp where encode = defaultEncode
+
+--------------------------------------------------------------------- Edit PickUp Location ---------------------------------------------------------- 
+data EditLocationRequest = EditLocationRequest EditLocationReq String
+
+newtype EditLocationReq = EditLocationReq {
+  destination :: EditLocation ,
+  origin :: EditLocation
+}
+
+newtype EditLocation = EditLocation {
```

Comment:
Same type already exist named `SearchReqLocation`

Rename to common name and reuse